app/code/Bulbulatory/Recommendations/view/frontend/layout/customer_account.xml

```diff
@@ -0,0 +1,13 @@
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+       <referenceBlock name="customer_account_navigation">
+            <block class="Magento\Framework\View\Element\Html\Link\Current" name="customer-account-navigation-custom">
```

Comment:
Pozycja _Polecenia_ w panelu klienta jest widoczna gdy moduł jest wyłączony w konfiguracji. Spróbuj zrobić tak, żeby pozycja była widoczna tylko wtedy gdy moduł poleceń jest aktywny (włączony w konfiguracji). Poczytaj o argumencie `ifconfig` - dzięki niemu będziesz mógł to zrobić.